bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx

```diff
@@ -162,22 +177,27 @@ export default class ZoomTool extends Component {
               role="button"
               label="-"
               icon="minus"
-              onClick={() => {}}
+              onClick={() => { }}
               disabled={(value <= minBound)}
               className={styles.prevSlide}
               hideLabel
             />
           </HoldButton>
         ),
         (
-          <span
+          <Button
             key="zoom-tool-2"
+            role="button"
```

Comment:
I noticed this in other spots in the code too - why do we specify `role="button"` to `Button`s? Don't they already have it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Why wouldn't the `role="button"` be set on the BaseButton?